client/src/components/EditProfile/EditProfileTab/EditProfileTab.tsx

```diff
@@ -55,7 +55,7 @@ export default function EditProfileTab(): JSX.Element {
   const userId = '619c1eb37a1e963a5b179c4b';
 
   useEffect(() => {
-    fetchProfile(userId).then((data) => setProfile(data.success.profile)); //get profileValues and set to profile state
+    fetchProfile('619c1eb37a1e963a5b179c4b').then((data) => setProfile(data.success.profile)); //get profileValues and set to profile state
```

Comment:
I accidentally committed this change. I faced some errors fetching the data since the helper function requires one of my pending PRs to be merged. Initially, I thought the issue was with my database, so I tried different hardcoded IDs to attempt to resolve it. Once that PR is merged, I will pass the userId to every component that needs it via a context.